Loop/Managers/Store Protocols/CarbStoreProtocol.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CarbStoreProtocol.swift
+//  Loop
+//
+//  Created by Anna Quinlan on 8/17/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+import HealthKit
+
+public protocol CarbStoreProtocol: AnyObject {
```

Comment:
For all of these stores, I've checked that all of the properties are needed in order for LDM to not error/function properly. I _could_ do more checking & casting into the desired store type, but I tried to minimize that so I'm not cluttering LDM